integrationtest/mapper/pojo/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedEmbeddedBaseIT.java

```diff
@@ -484,6 +484,79 @@ public IndexedEmbeddedLevel1 getLevel1() {
 		);
 	}
 
+	@Test
+	public void maxDepth_differentDepths() {
```

Comment:
Could you please add this test to a new class in the `nonregression` package? Also please annotate with `@TestForIssue`.

It helps to understand why a test was added in the first place, if it ever starts failing again.

Reply:
There was also a not used variable...